client/me/developer/style.scss

```diff
@@ -84,3 +84,112 @@
 	}
 }
 
+.developer-survey-notice {
+	position: fixed;
+	left: 0;
+	top: 0;
+	height: 100%;
+	width: 100%;
+	z-index: 1000;
+
+	.developer-survey-notice__backdrop {
+		background: var(--studio-black);
+		opacity: 0.2;
+		position: absolute;
+		left: 0;
+		top: 0;
+		width: 100%;
+		height: 100%;
+		cursor: default;
+	}
+
+	.developer-survey-notice__popup {
+		position: absolute;
+		right: 25px;
+		bottom: 25px;
+		width: 416px;
+		max-width: calc(100% - 50px);
+		z-index: 2;
+		border-radius: 2px;
+		box-shadow: 0 3px 1px 0 rgba(0, 0, 0, 0.04), 0 3px 8px 0 rgba(0, 0, 0, 0.12);
+		overflow: hidden;
+		font-family: $font-sf-pro-text;
+
+		.developer-survey-notice__popup-head {
+			background: #0675c4;
+			border-bottom: 1px solid #f6f7f7;
+			height: 56px;
+			padding: 0 14px 0 16px;
+			display: flex;
+			align-items: center;
+			justify-content: space-between;
+
+			.developer-survey-notice__popup-head-title {
+				color: var(--studio-white);
+				font-size: rem(14px);
+				font-weight: 500;
+				line-height: 20px;
+				letter-spacing: -0.15px;
+			}
+
+			.developer-survey-notice__popup-head-close svg {
+				fill: var(--studio-white);
+			}
+		}
+
+		.developer-survey-notice__popup-img {
+			background: #0675c4;
+			height: 241px;
```

Comment:
```suggestion
			height: 0;
			padding-bottom: 57.9%;
```

On mobile, we have extra padding underneath the image because of the set height. The suggested change sets the height based on the aspect-ratio of the image.

| Before | After |
| ------ | ------ |
|<img width="396" alt="Screen Shot 2024-02-08 at 10 26 42 AM" src="https://github.com/Automattic/wp-calypso/assets/104910361/7fd1f972-e588-48a2-9aa7-132d5ced68c4">|<img width="401" alt="Screen Shot 2024-02-08 at 10 28 07 AM" src="https://github.com/Automattic/wp-calypso/assets/104910361/ceb0ef3a-f3d8-44b4-b80e-7d7039bb98a1">|